sdk/python/tests/compiler/testdata/resourceop_basic.yaml

```diff
@@ -67,7 +67,7 @@ spec:
             is failure.
           name: failure-condition
           type: string
-        - default: aipipeline/kubectl-wrapper:0.8.0
+        - default: aipipeline/kubectl-wrapper:0.8.1
```

Comment:
this new version tag `0.8.1` of the `kubectl-wrapper` got generated by the compiler. I did not create/push a new `aipipeline/kubectl-wrapper:0.8.1` image, though

Reply:
@Tomcli -- which of two options should we go with here?

- [ ] to avoid this automatic version bump of the `kubectl-wrapper` image, should we keep the SDK version at `0.8.0` in our repo (`sdk/python/kfp_tekton/__init__.py`), and only create the `0.8.1` version in PyPI? 
- [ ] or generate/push a new `0.8.1` image of the `kubectl-wrapper`

Reply:
I can build a 0.8.1 image and push it. There's no code changes so it's still the same image. I can't do it with our internal CI since the internal CI is building it based on the Github release version.

Reply:
Thanks Tommy